pkg/agent/openflow/pipeline.go

```diff
@@ -218,6 +218,7 @@ const (
 	// cnpDropConjunctionIDReg reuses reg3 which will also be used for storing endpoint IP to store the rule ID. Since
 	// the service selection will finish when a packet hitting NetworkPolicy related rules, there is no conflict.
 	cnpDropConjunctionIDReg regType = 3
+	endpointIPv6XXReg       regType = 3 // xxReg3 will occupy 12-15 regs
```

Comment:
I think we'd better introduce a new type here to avoid confusion. @wenyingd What do you think?

Reply:
Could be. But then we risk someone reusing regs 12-15 here, overwriting the xxreg.

Reply:
I'm not sure what "xxReg3 will occupy 12-15 regs" here? xxReg3 will occupy reg12-reg15?

Reply:
Indeed. Will change the comment.